vision/module/module_bounding.py

```diff
@@ -0,0 +1,46 @@
+"""
+This file contains getModuleBounds, which will get a list of four vertices in the module.
+"""
+
+import numpy as np
+
+MODULE_HEIGHT = 76.2  # mm
+MODULE_WIDTH = 50.8  # mm
+VERTICAL_FOV = 57  # degrees
+HORIZONTAL_FOV = 86  # degrees
+
+PADDING_CONSTANT = .85
+
+def getModuleBounds(color, center, depth):
+    """
+    getModuleBounds will find four coordinates within the module that will be used to create a BoundingBox.
+
+    Parameters
+    ----------
+    color - ndarray - the color image
```

Comment:
For clarity, lets just pass the dimensions of the color image instead of the whole thing.